src/pages/Home.jsx

```diff
@@ -1,9 +1,9 @@
 import ArticleList from "../components/article/ArticleList";
 
-function Home() {
+function Home(setArticle) {
```

Comment:
What's the point in this component? You might as well just show an article list on the initial '/' path.